x-pack/metricbeat/module/azure/_meta/kibana/7/dashboard/Metricbeat-azure-blob-storage-overview.json

```diff
@@ -0,0 +1,1299 @@
+          "type": "index-pattern"
+        },
+        {
+          "id": "e4b25ee0-32f6-11ea-a83e-25b8612d00cc",
+          "name": "panel_0",
+          "type": "visualization"
+        },
+        {
+          "id": "40dbc0d0-32e3-11ea-a83e-25b8612d00cc",
+          "name": "panel_1",
+          "type": "visualization"
+        },
+        {
+          "id": "17ee2920-3391-11ea-a83e-25b8612d00cc",
+          "name": "panel_2",
+          "type": "visualization"
+        },
+        {
+          "id": "55936920-3391-11ea-a83e-25b8612d00cc",
+          "name": "panel_3",
+          "type": "visualization"
+        },
+        {
+          "id": "553f9320-32e9-11ea-a83e-25b8612d00cc",
+          "name": "panel_4",
+          "type": "visualization"
+        },
+        {
+          "id": "acced050-32d1-11ea-a83e-25b8612d00cc",
+          "name": "panel_5",
+          "type": "visualization"
+        },
+        {
+          "id": "81f16b40-32ea-11ea-a83e-25b8612d00cc",
+          "name": "panel_6",
+          "type": "visualization"
+        },
+        {
+          "id": "685fbeb0-32eb-11ea-a83e-25b8612d00cc",
+          "name": "panel_7",
+          "type": "visualization"
+        },
+        {
+          "id": "9e991b10-3391-11ea-a83e-25b8612d00cc",
+          "name": "panel_8",
+          "type": "visualization"
+        },
+        {
+          "id": "599c62c0-32d7-11ea-a83e-25b8612d00cc",
+          "name": "panel_9",
+          "type": "visualization"
+        },
+        {
+          "id": "4eaef260-32e1-11ea-a83e-25b8612d00cc",
+          "name": "panel_10",
+          "type": "visualization"
+        }
+      ],
+      "type": "dashboard",
+      "updated_at": "2020-01-10T10:16:36.469Z",
+      "version": "WzMxMjYsMV0="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "Navigation Blob Storage Overview [Metricbeat Azure]",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [],
+          "params": {
+            "fontSize": 10,
+            "markdown": "### Azure Storage\n\n[Overview](#/dashboard/1a151f80-32db-11ea-a83e-25b8612d00cc) | [**Blob**](#/dashboard/b165ef60-32f7-11ea-a83e-25b8612d00cc) | [File](#/dashboard/dff7a080-32f7-11ea-a83e-25b8612d00cc) | [Table](#/dashboard/ff2fe020-32f7-11ea-a83e-25b8612d00cc) | [Queue](#/dashboard/10efa340-32f8-11ea-a83e-25b8612d00cc) ",
+            "openLinksInNewTab": false
+          },
+          "title": "Navigation Blob Storage Overview [Metricbeat Azure]",
+          "type": "markdown"
+        }
+      },
+      "id": "e4b25ee0-32f6-11ea-a83e-25b8612d00cc",
+      "migrationVersion": {
+        "visualization": "7.4.2"
```

Comment:
got:
```
error loading C:\Users\Mariana\WORK\GO\src\github.com\elastic\beats\x-pack\metricbeat\kibana\default\dashboard\Metricbeat-azure-file-storage-overview.json: 422 Unprocessable Entity. Response: {"statusCode":422,"error":"Unprocessable Entity","message":"D
ocument \"f18a7cb0-32f6-11ea-a83e-25b8612d00cc\" has property \"visualization\" which belongs to a more recent version of Kibana (7.4.2)."}
```
when I ran `metricbeat --setup`
Not sure if this happening only in my case or a lower version for the visualizations is necessary here. Replaced `7.4.2` in all dashboards files with `7.3.1` and I got no problems from there

Reply:
did it happen only for the particular dashboard?

Reply:
I had the same error with all of them